src/libraries/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -32,129 +32,140 @@ protected override string GetExistingItem(bool readOnly = false)
-                    ((path) => File.GetCreationTime(path)),
+                    SetCreationTime,
+                    GetCreationTime,
                     DateTimeKind.Local);
                 yield return TimeFunction.Create(
-                    ((path, time) => File.SetCreationTimeUtc(path, time)),
-                    ((path) => File.GetCreationTimeUtc(path)),
+                    SetCreationTimeUtc,
+                    GetCreationTimeUtc,
                     DateTimeKind.Unspecified);
                 yield return TimeFunction.Create(
-                    ((path, time) => File.SetCreationTimeUtc(path, time)),
-                    ((path) => File.GetCreationTimeUtc(path)),
+                    SetCreationTimeUtc,
+                    GetCreationTimeUtc,
                     DateTimeKind.Utc);
             }
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastAccessTime(path, time)),
-                ((path) => File.GetLastAccessTime(path)),
+                SetLastAccessTime,
+                GetLastAccessTime,
                 DateTimeKind.Local);
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastAccessTimeUtc(path, time)),
-                ((path) => File.GetLastAccessTimeUtc(path)),
+                SetLastAccessTimeUtc,
+                GetLastAccessTimeUtc,
                 DateTimeKind.Unspecified);
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastAccessTimeUtc(path, time)),
-                ((path) => File.GetLastAccessTimeUtc(path)),
+                SetLastAccessTimeUtc,
+                GetLastAccessTimeUtc,
                 DateTimeKind.Utc);
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastWriteTime(path, time)),
-                ((path) => File.GetLastWriteTime(path)),
+                SetLastWriteTime,
+                GetLastWriteTime,
                 DateTimeKind.Local);
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastWriteTimeUtc(path, time)),
-                ((path) => File.GetLastWriteTimeUtc(path)),
+                SetLastWriteTimeUtc,
+                GetLastWriteTimeUtc,
                 DateTimeKind.Unspecified);
             yield return TimeFunction.Create(
-                ((path, time) => File.SetLastWriteTimeUtc(path, time)),
-                ((path) => File.GetLastWriteTimeUtc(path)),
+                SetLastWriteTimeUtc,
+                GetLastWriteTimeUtc,
                 DateTimeKind.Utc);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotInAppContainer))] // Can't read root in appcontainer
-        [PlatformSpecific(TestPlatforms.Windows)]
-        public void PageFileHasTimes()
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void BirthTimeIsNotNewerThanLowestOfAccessModifiedTimes()
```

Comment:
Same as earlier comment. Because this was moved in the file, it is harder to review the changes.